spec/models/form_profile_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'support/attr_encrypted_matcher'
+
+RSpec.describe FormProfile, type: :model do
+  let(:user) { build(:loa3_user) }
+  let(:expected) do
+    {
+      'veteranFullName' => {
+        'first' => 'Abraham',
```

Comment:
Nitpicky, but could the data within this expected hash be populated from their original source so as not to duplicate it?  For example
```ruby
'first' => user.first_name,
```